toil_orchestrator/admin.py

```diff
@@ -1,44 +1,62 @@
 from django.contrib import admin
-from .models import Job, CommandLineToolJob
+from .models import Job, CommandLineToolJob, Status
 from toil_orchestrator.tasks import cleanup_folders
 from django.contrib import messages
 
 
+class StatusFilter(admin.SimpleListFilter):
+    title = 'Status'
+    parameter_name = 'status'
+
+    def lookups(self, request, model_admin):
+        filters = {k:v for (k, v) in request.GET.items() if "range" not in k and "status" not in k
+                   and "q" not in k and "p" not in k}
```

Comment:
could you help me understand what `range` and `status` and `q` and `p` represent here?